docs/modules/operation/pages/events/event-debugging.adoc

```diff
@@ -93,7 +109,6 @@ For example, `-p 'url \http://www.opennms.com/'`
 
 | -V (--version)
 | Print version and exit successfully.
-
 |===
 
 Note that parameter descriptions use the percentage sign (%) as a delimiter to mark a replacement variable.
```

Comment:
Line 115, replacement tokens link does not go to the right place.
Line 117, replace colon with comma after "For example,"